DOCKER.md

```diff
@@ -0,0 +1,34 @@
+# Viewing the AtoM docs with Docker Compose
+
+This [Access to Memory](https://www.accesstomemory.org/) documentation can be
+generated and viewed locally in a web browser with the provided Docker Compose
+configuration.  Viewing the documentation locally can be very helpful when
+making updates to quickly check for syntax or content errors.
+
+## Prerequistes
+
+**These instructions require a working installation of the Docker Engine**
+
+See [Docker Engine Install](https://docs.docker.com/engine/install/) for
+instructions on installing Docker in your operating system.
+
+## Building a viewing the documentation with Docker Compose
+
+1. In the atom-docs directory run `docker-compose up -d` to build and start
+   the atom-docs Docker container
+1. Once the container is built, open a web browser and navigate to
+  `http://localhost:8100`
+
+![Screenshot of AtoM docs in Docker](images/atom_docs_in_docker.png)
```

Comment:
Maybe for another iteration, but it would be great to see the actual theme we use in the docs site.

Reply:
Agreed. :)